docs/rc.xml.all

```diff
@@ -69,23 +69,27 @@
     See man labwc-actions for further information.
 
     Workspaces can be configured like this:
-    <desktops>
```

Comment:
I think it would be better to provide two examples here. One using number and the other (as-is) using names.name 

…because using both together doesn’t really make sense. It should be one or the other.